content/hi/continuous_integration.md

```diff
@@ -0,0 +1,17 @@
+---
+title: निरंतर इंटीग्रेशन
```

Comment:
We can probably have `एकीकरण` in the title  for integration.

Reply:
Ok thank you. I will do it